### PR TITLE
Fix snackbar in registration, login, forgot-password, profile

### PR DESCRIPTION
**Pull request by naberalgamma**

### CODE REVIEW

**Review by obozhko-dev — COMMENTED** [review]
In different components different height of message block 
image
image
image

**Comment by naberalgamma** [comment]
fixed snackbar margin

File: frontend/src/components/Login/Login.tsx [hunk 1]
```diff
@@ -113,12 +113,17 @@ function Login() {
...
                 >
-                  <Alert onClose={handleCloseNotification} severity="error">
+                  <Alert
+                    onClose={handleCloseNotification}
+                    severity="error"
+                    sx={{ mt: '1vh' }}
+                  >
                     {notification}
                   </Alert>
                 </Snackbar>
```
Comment (6mahdapihka9) [inline frontend/src/components/Login/Login.tsx:129]: I think you should create a universal component like StyledSnackBar or something and reuse it in all HOCs, so styles won't repeat in each usage.

Just reinvoke it with the needed type and message in Login, Registration and ForgotPassword

File: frontend/src/components/Registration/Registration.tsx [hunk 2]
```diff
@@ -82,17 +98,32 @@ function Registration() {
...
-                  <Alert severity="error">{errorMessage}</Alert>
+                  <Alert
+                    onClose={handleCloseNotification}
+                    severity="success"
+                    sx={{ mt: '1vh' }}
+                  >
+                    {t('registration.regisrationSuccess')}
+                  </Alert>
                 </Snackbar>
                 <Controller
```
Comment (Yojuks) [inline frontend/src/components/Registration/Registration.tsx:128]: Screenshot 2022-05-30 at 14 02 49
should correct this problem

**Review by Yojuks — APPROVED** [review]

**Review by Yojuks — APPROVED** [review]

**Review by 6mahdapihka9 — APPROVED** [review]

**Pull request merged**